### DISCUSSION TITLE
Vaadin Extension visible twice in Dev UI

### DISCUSSION THREAD

**Discussion by simasch** [quarkusio/quarkus#40701] [Community]
I see the Vaadin extension twice in the Dev UI how can I find out why this happens?

<img width="1733" alt="image" src="https://github.com/quarkusio/quarkus/assets/593352/fd852ce8-d5bd-41da-9dc5-beb2bb221e20">

**Comment by phillip-kruger** [comment]
That is weird. The extension cards are build from the Metadata (quarkus-extension.yaml) in the runtime module. Maybe there are two ? Where is the code for the extension ?

**Comment by simasch** [comment]
This is the extension: https://quarkus.io/extensions/com.vaadin/vaadin-quarkus-extension/

I also checked the source code but couldn't find a problem:
https://github.com/vaadin/quarkus

**Comment by phillip-kruger** [comment]
Seems like there are two jars that contains meta data for Quarkus:

- com.vaadin:vaadin-quarkus-extension::jar:24.3.12
- com.vaadin:vaadin-quarkus::jar:2.0.1

**Comment by simasch** [comment]
Thanks, @phillip-kruger. I'll report that in the extension repo.
